config/environments/development.rb

```diff
@@ -15,4 +15,8 @@
   config.smoke_test_email_local_part = ENV['SMOKE_TEST_EMAIL_LOCAL_PART']
   config.smoke_test_email_domain = ENV['SMOKE_TEST_EMAIL_DOMAIN']
   config.session_expire_after = 24.hours
+
+  config.prison_data = YAML.load_file(
```

Comment:
@dan-palmer No, because models aren't yet initialised when application.rb loads.  Works fine in an initialiser, though.